spec/cucumber/formatter/html_spec.rb

```diff
@@ -25,9 +25,9 @@ module Formatter
       end
 
       it 'does not raise an error when visiting a blank feature name' do
-        expect(-> {
+        expect(-> do
           @formatter.feature_name('Feature', '')
-        }).not_to raise_error
+        end).not_to raise_error
```

Comment:
One way to get this to read a little easier is to try keeping the "stabby lambda" and having it on 1 line:

```ruby
expect(->{ @formatter.feature_name('Feature', '') }).not_to raise_error
```

Reply:
All set here!